config/data_centers/data_centers.json

```diff
@@ -898,5 +898,145 @@
     "status": "operational",
     "source": "https://docs.aws.amazon.com/global-infrastructure/latest/regions/aws-regions.html",
     "operationalSince": null
+  },
+  "ovh-us-hillsboro": {
+    "provider": "ovh",
+    "displayName": "Hillsboro",
+    "zoneKey": "US",
```

Comment:
Zone key should also be more specific and not use a aggregate.

Reply:
I think Victor meant the `zoneKey` that tells Electricity Map the electrical grid region where the data centre is located.